docs/components/servo/_index.md

```diff
@@ -339,6 +339,14 @@ You can find additional assistance in the [Troubleshooting section](/appendix/tr
 
 {{< snippet "social.md" >}}
 
+## Test the servo
+
+After setting up and establishing the connection for your servo, navigate to the [Control tab](https://docs.viam.com/manage/fleet/robots/#control) and select the dropdown panel dedicated to the motor.
+
+![The servo component in the control tab](assets/components/servo/servo-control-tab.png)
+
+Use the buttons to move the servo motor to the desired angle.
+
```

Comment:
Ok - I think if you remove this bit, which we don't want because we only want to add the information on testing just underneath when people configure their servos, then this will build correctly. You're currently getting a build error which you can see [here](https://github.com/viamrobotics/docs/actions/runs/6017599821/job/16324118150?pr=1717).

```suggestion
```